tests/installation/partitioning_togglehome.pm

```diff
@@ -16,17 +16,12 @@ use base "y2logsstep";
 use testapi;
 use installation_user_settings;
 use version_utils 'is_storage_ng';
+use partition_setup 'unselect_xen_pv_cdrom';
 
 sub run {
     wait_screen_change { send_key(is_storage_ng() ? 'alt-g' : 'alt-d') };    # open proposal settings
```

Comment:
And JFYI, we started redefining some cmd keys in case of storage-ng in partitioning test module to avoid such switches. So maybe you could use that in future commits =)

Reply:
Thx :), note taken.